AppDelegate+Injection.swift

```diff
@@ -255,18 +256,18 @@ extension Resolver: ResolverRegistering {
                 }
             }
         }
-        
+
         private var children: [FeatureFlagProvider]
-        
+
         init(children: [FeatureFlagProvider]) {
             self.children = children
         }
-        
+
         func isFeatureOn(_ feature: FeatureFlag) -> Bool? {
             children.compactMap { $0.isFeatureOn(feature) }.first
         }
     }
-    
+
```

Comment:
would it be possible to revert unneeded changes here? I think this file is untouched except whitespaces and this is a pretty conflict-prone one